Packs/Netskope/Integrations/NetskopeEventCollector/NetskopeEventCollector.yml

```diff
@@ -17,28 +17,19 @@ configuration:
   required: true
   type: 9
   section: Connect
-- additionalinfo: The API version to use (v1 or v2).
-  defaultvalue: v2
-  display: API Version
-  name: api_version
-  type: 15
-  section: Connect
-  options:
-  - v1
-  - v2
-  required: false
 - additionalinfo: 'The first fetch is limited to the last 50,000 events (for each event type).'
```

Comment:
I don't think this is necessary anymore

Reply:
The entire arg is actually not needed anymore